erpnext/www/book-appointment/index.html

```diff
@@ -0,0 +1,66 @@
+{% extends "templates/web.html" %}
```

Comment:
UI  Recommendations

## Page 1 

- [ ] Spacing between the two inputs can be reduced in mobile
- [ ] The inputs can be made full width in mobile (Making them equal width at the same time)
- [ ] The padding for the slots is too tight... That extra space on the right and left is unnecessary 

 <details>
  <summary>Screenshot</summary>
  <img src="https://user-images.githubusercontent.com/18097732/68562408-7c35c100-046f-11ea-883f-6194470b24fe.png">
</details> 

---

- [ ] Inconsistent width of the timing cards to be fixed... (Use [CSS](https://css-tricks.com/snippets/css/complete-guide-grid/) grids if you are looking for some nice CSS)

<details>
  <summary>Screenshot</summary>
<img width="313" alt="Screenshot 2019-11-11 at 10 41 29 AM" src="https://user-images.githubusercontent.com/18097732/68562487-ddf62b00-046f-11ea-89ea-01676329bab2.png">
</details> 

## Page 2

- [ ] The subtitle can be muted
<details>
  <summary>Screenshot</summary>
<img width="313" alt="Screenshot 2019-11-11 at 10 41 29 AM" src="https://user-images.githubusercontent.com/18097732/68562630-7f7d7c80-0470-11ea-8336-a0f6906e35f6.png">
</details> 

- [ ] Spacing between "Go Back" and "Submit"
- [ ] While in desktop, Submit can be on the right... In Mobile you'd want Submit button to be on top.
<details>
  <summary>Screenshot</summary>
<img width="313" alt="Screenshot 2019-11-11 at 10 41 29 AM" src="https://user-images.githubusercontent.com/18097732/68562717-dc793280-0470-11ea-8ed5-292771ae2d13.png">
</details> 

## Page 3

- [ ] Unclear error states when form is partially filled
- [ ] No success state, Once the appointment is created the user just stays on the last form, maybe redirect the customer to a success link (configurable in appointment settings page) or a JS rendered page (just like the form and date picker) which has a custom message which is configured by the ERPNext User
<details>
  <summary>Screenshot</summary>
<img width="313" alt="Screenshot 2019-11-11 at 10 41 29 AM" src="https://user-images.githubusercontent.com/18097732/68563268-fddb1e00-0472-11ea-8aa3-8fb79127af04.png">
</details>

Reply:
UI fixes applied.
Also added a configurable redirect on success.